lib/screens/speaker/speaker.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:ecellapp/core/utils/logger.dart';
+import 'package:ecellapp/models/speaker.dart';
+import 'package:ecellapp/screens/speaker/cubit/speaker_cubit.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class SpeakerScreen extends StatefulWidget {
```

Comment:
since there are no setState function calls required, this should be a stateless widget

Reply:
I applied StatefulWrapper is it still required now?

Reply:
The reason for using stateful wrapper was so that we can have an init function in a stateless widget. You still have to make it stateless.